script/shard_spec_repo

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env ruby
+
+require 'bundler/inline'
+
+gemfile do
+  gem 'cocoapods', '>= 1.1.0.rc.1'
+  gem 'cocoapods-repo-shard'
+
+  gem 'sequel'
+end
+
+Pod::Config.instance.verbose = true
+
+db = Sequel.connect(ENV['DATABASE_URL'])
+
+master = Pod::SourcesManager.master.first
+prefix_lengths = [1, 1, 1]
+
+Pod::Executable.execute_command(:heroku, %w(config:set TRUNK_APP_PUSH_ALLOWED=false), true)
+
+Pod::Command.run(%w(repo update master))
+Pod::Command.run(%W(repo shard master --lengths=#{prefix_lengths.join(',')}))
+
+shard_sha = Dir.chdir(master.repo) { Pod::Executable.execute_command(:git, %w(rev-parse HEAD), true).strip }
+
+# creating from scratch to avoid re-serializing any stale data
+Pod::Source::Metadata.from_file(master.metadata_path).tap do |metadata|
+  metadata.minimum_cocoapods_version = Pod::Version.new('1.0.0.beta.6')
```

Comment:
_probably_ better to have that as a non-beta? I know that's when we added support, but probably safer